kinode/src/kernel/mod.rs

```diff
@@ -826,6 +845,82 @@ pub async fn kernel(
         })
         .await
         .expect("fatal: kernel event loop died");
+    // sending hard coded routers into networking for bootstrapped rpc
+    let kns_updates = vec![
```

Comment:
We need to move this elsewhere. Possibly into a dedicated JSON file that gets parsed in main? Not here though

Reply:
moved it into kinode/default-router.json

it attempts to read home/.routers first though. and this is only passed to net:distro:sys, so if the user has any newer state for these routers in the indexer, they should be overwritten correctly on boot